src/inspect_ai/tool/_tools/_web_browser/_web_browser.py

```diff
@@ -11,43 +12,59 @@
 from inspect_ai.util._store import store
 
 
-def web_browser(interactive: bool = True) -> list[Tool]:
```

Comment:
For this sort of thing our style is to use typed `Literal`, so something like `format: Literal["at", "html"]`